permissions/index.js

```diff
@@ -1,40 +1,75 @@
 const User = require('./user');
 const { SafeRouter, publicEndpoint } = require('./safe-router');
-const { applyPermissions: applyZmPermissions } = require('./rules/zm');
-const { applyPermissions: applyZaPermissions } = require('./rules/za');
-const { applyPermissions: applyEtPermissions } = require('./rules/et');
-const { applyPermissions: applyMwPermissions } = require('./rules/mw');
-const { applyPermissions: applyBrPermissions } = require('./rules/br');
-const { applyPermissions: applyInPermissions } = require('./rules/in');
-const { applyPermissions: applyJmPermissions } = require('./rules/jm');
-const { applyPermissions: applyCaPermissions } = require('./rules/ca');
-const { applyPermissions: applyOpenPermissions } = require('./rules/open');
+const { setupCanForRules } = require('./setupCanForRules');
+
+const openRules = require('./rules/open.json');
+const brRules = require('./rules/br.json');
+const caRules = require('./rules/ca.json');
+const etRules = require('./rules/et.json');
+const inRules = require('./rules/in.json');
+const jmRules = require('./rules/jm.json');
+const mwRules = require('./rules/mw.json');
+const zaRules = require('./rules/za.json');
+const zmRules = require('./rules/zm.json');
+
 const { canEditCase, canViewPostSurvey } = require('./middlewares');
 
-const applyPermissions = {
-  zm: applyZmPermissions,
-  za: applyZaPermissions,
-  et: applyEtPermissions,
-  mw: applyMwPermissions,
-  br: applyBrPermissions,
-  in: applyInPermissions,
-  jm: applyJmPermissions,
-  ca: applyCaPermissions,
+// TODO: maybe factor out so it's easier to build the API that retrieves this to the frontend
+const rulesMap = {
+  br: brRules,
+  ca: caRules,
+  et: etRules,
+  in: inRules,
+  jm: jmRules,
+  mw: mwRules,
+  za: zaRules,
+  zm: zmRules,
+};
+
+/**
+ * @type {{ [key in keyof typeof rulesMap]: ReturnType<typeof setupCanForRules> }}
+ */
+const initializedCanMap = Object.entries(rulesMap).reduce((accum, [key, rules]) => {
+  const can = setupCanForRules(rules);
+  return {
+    ...accum,
+    [key]: can,
+  };
+}, {});
```

Comment:
When the server starts, this object is constructed with all the existing permissions files processed by `setupCanForRules`. The result of `setupCanForRules` is either a `can` function with the configuration as provided (if the file is valid) or an error string if something went wrong. I think that with this behavior is easy to spot potential issues with a single config file without affecting the rest (you can `console.log` this object while working locally, if the set of rules being edited is a function, that's fine, if not the string will tell you what's wrong).